CSS: Remove bad practice (#35363)
mdn/content: files/en-us/web/css/transition/index.md

### Basic example

> In this example, when the user hovers over the element, there is a one-second delay before a four-second `background-color` transition occurs.

Author: ```suggestion
In this example, when the user hovers over the element, there is a half-second (`500ms`) delay before a two-second `background-color` transition occurs.
```

```css
.target {
  font-size: 2rem;
  background-color: palegoldenrod;

>   transition: background-color 4s 1s;

Author: ```suggestion
  transition: background-color 2s 500ms;
```

}

.target:hover {

>   background-color: lime;

Reviewer: I can barely notice a difference between the before and after transition colors. Maybe change it to colors with a more striking difference, for example, `darkorange` and `yellow`?

Also maybe reduce the duration from `4s` to `2s`, so the change is a bit more perceivable?

Author: agree. changed. Thanks.

>   background-color: lime;

Author: ```suggestion
  background-color: darkorange;
```
